### ISSUE TITLE
Add Relative Path for the AppShell

### ISSUE THREAD

**Issue by jdevillard** [smapiot/Piral.Blazor.Server#2]
# New Feature Proposal

Add capability to Emulator to load app shell from a relative path to the Pilet.
For now, we can use an app shell using an absolute path.

## Description

We can add an attribute to `<AppShell />` like <AppShell relative="true">../../path/to/shell</AppShell>

## Background

This will allow faster dev/test on local environment.

**Comment by FlorianRappl** [comment]
> For now, we can use an app shell using an absolute path.

That is not true. You can use NuGet package paths. This is also the recommended way.

**Comment by FlorianRappl** [comment]
> So when working on a Solution with Shell and Pilet, we have to pack the nuget and copy it to the %userprofile%/.nuget/packages folder, no ?

I don't know. Well, at some point you need to pack and distribute; when that is would be up to you. The ideal situation is that a local shell is detected and just used.
